### PR TITLE
update rules-docker, and add a python base image

### PR DESCRIPTION
**Pull request by krzyzacy**
split up from https://github.com/kubernetes/test-infra/pull/6835

/area bazel
/assign @BenTheElder @ixdy

### CODE REVIEW

File: WORKSPACE [hunk 1]
```diff
@@ -76,6 +76,14 @@ docker_pull(
     repository = "k8s-prow/git",
 )
 
+docker_pull(
+    name = "python",
+    digest = "sha256:3f60d995e4aa7d7a5e4783c185e13d53e124ef0ad87bc1492f869cbeead331ee",
+    registry = "index.docker.io",
+    repository = "library/python",
+    tag = "2.7.14-alpine3.7",
```
Comment (ixdy) [inline WORKSPACE]: does this version of the python image contain git?

**Comment by ixdy** [comment]
oh, I think you're broken by https://github.com/bazelbuild/rules_docker/commit/15a63369d9a7aa7aee2bffc3bb4a76fb4a629bc4, which hasn't yet been fixed in rules_k8s. @mattmoor

File: WORKSPACE (hunk 1 above)
Comment (krzyzacy) [reply WORKSPACE]: nope :cry: good catch

**Comment by krzyzacy** [comment]
rolled back a bit to https://github.com/bazelbuild/rules_k8s/commit/3a90662f22e3e38ee5553a695e1215a4b0aeb514

**Comment by krzyzacy** [comment]
okay... looks green now

let me verify this works for my fork

**Comment by mattmoor** [comment]
@ixdy https://github.com/bazelbuild/rules_k8s/pull/118

**Comment by ixdy** [comment]
@mattmoor thanks for being my interface to the github search API :)

**Comment by krzyzacy** [comment]
/hold cancel
:crossed_fingers:

**Pull request merged**